src/libraries/System.IO.Compression/src/System/IO/Compression/ZipArchiveEntry.Windows.cs

```diff
@@ -7,6 +7,18 @@ public partial class ZipArchiveEntry
     {
         internal const ZipVersionMadeByPlatform CurrentZipPlatform = ZipVersionMadeByPlatform.Windows;
 
+        /// <summary>
+        /// The default external file attributes are used to support zip archives on multiple platforms.
+        /// Since Windows doesn't use file permissions, there's no default value needed.
+        /// </summary>
+        internal const uint DefaultFileExternalAttributes = 0;
```

Comment:
I'm curious. What happens when you extract the `ZipVersionMadeByPlatform.Windows` file on Unix using the system tools. What permissions do you get?

Reply:
https://github.com/dotnet/runtime/blob/ddc4f95432facf2deebfa85bad7143469dca0129/src/libraries/System.IO.Compression.ZipFile/src/System/IO/Compression/ZipFileExtensions.ZipArchiveEntry.Extract.Unix.cs#L20-L26

Reply:
@tmds it took me a while to test this: was trying to understand how to set up the dev environment on my old windows laptop. 

Here's the result unpacked on a mac machine:
```
 derwasp@xxxxx  ~/Downloads/ziptest > unzip archive_created_on_win.zip -d win_arch
Archive:  archive_created_on_win.zip
   creating: win_arch/folder/
  inflating: win_arch/folder/inmemory_file.txt
 derwasp@xxxxx  ~/Downloads/ziptest > cd win_arch
 derwasp@xxxxx  ~/Downloads/ziptest/win_arch > ls -la
total 0
drwxr-xr-x   3 derwasp  staff   96 Jun 24 22:49 .
drwxr-xr-x  14 derwasp  staff  448 Jun 24 22:49 ..
drwxr-xr-x   3 derwasp  staff   96 Jun 24 13:45 folder
 derwasp@xxxxx  ~/Downloads/ziptest/win_arch > cd folder
 derwasp@xxxxx  ~/Downloads/ziptest/win_arch/folder > ls -la
total 8
drwxr-xr-x  3 derwasp  staff  96 Jun 24 13:45 .
drwxr-xr-x  3 derwasp  staff  96 Jun 24 22:49 ..
-rw-r--r--  1 derwasp  staff  56 Jun 24 13:45 inmemory_file.txt
```

I also verified that the default values are not set like so:

```
var arch = "/Users/derwasp/Downloads/ziptest/archive_created_on_win.zip";

await using (var fileStream = new FileStream(arch, FileMode.Open))
using (var archive = new ZipArchive(fileStream, ZipArchiveMode.Read, true))
{
    foreach (var e in archive.Entries)
    {
        Console.WriteLine($"{e.FullName}, \t\t attributes = {(e.ExternalAttributes >> 16)}");

    }
}
```

And this is the output:

```
folder/,                 attributes = 0
folder/inmemory_file.txt,                attributes = 0
```

So the `unzip` utility does it the way that we expected: `644` for the files and `755` for the folders.